sumologic/async.py

```diff
@@ -0,0 +1,162 @@
+from copy import copy
+import json
+import logging
+import asyncio
+import aiohttp
+
+class SumoLogic(object):
+
+    def __init__(self, accessId, accessKey, endpoint=None, cookieFile='cookies.txt'):
```

Comment:
there are a couple of rough edges here:
- [ ] cookies have to be added back if they are really needed
- [ ] remove explicitly setting the read timeout to infinity. I don't why but while fetching results for some search jobs a frequently got TimeOutErrors. I think this should be configurable at least.